plugins/package-managers/maven/src/main/kotlin/utils/MavenParsers.kt

```diff
@@ -160,7 +160,7 @@ internal fun parseLicenses(mavenProject: MavenProject): Set<String> =
 internal fun parseVcsInfo(project: MavenProject): VcsInfo =
     parseScm(getOriginalScm(project), project.artifact?.toString().orEmpty())
 
-internal fun parseScm(scm: Scm?, artifactId: String): VcsInfo {
+internal fun parseScm(scm: Scm?, artifactId: String, subPath: String = ""): VcsInfo {
```

Comment:
Nit: Why "subPath" instead of just "path"?

Reply:
The variable `path` is already used in the function, in line 176: `val (user, host, path) = match.destructured`. This introduces a hard to detect shading problem (guess why I noticed). If you prefer, I can rename the function parameter from `subPath` to `path` and rename the variable, maybe to `urlPath`?

Reply:
I'd instead rename `subPath` to e.g. `vcsPath` or `scmPath` then.

Reply:
Renamed to `vcsPath`.